cmd/kflex/ctx/ctx.go

```diff
@@ -84,9 +85,15 @@ func (c *CPCtx) Context(chattyStatus, failIfNone bool) {
 				fmt.Fprintf(os.Stderr, "Error loading kubeconfig context from server: %s\n", err)
 				os.Exit(1)
 			}
-			if err = kubeconfig.SwitchContext(kconf, c.Name); err != nil {
-				fmt.Fprintf(os.Stderr, "Error switching kubeconfig context after loading from server: %s\n", err)
-				os.Exit(1)
+			// context exists only for CPs that are not of type host
+			if c.Type != tenancyv1alpha1.ControlPlaneTypeHost {
+				if err = kubeconfig.SwitchContext(kconf, c.Name); err != nil {
+					fmt.Fprintf(os.Stderr, "Error switching kubeconfig context after loading from server: %s\n", err)
+					os.Exit(1)
+				}
+			} else {
+				fmt.Fprintf(os.Stderr, "control plane %s is of type 'host', using hosting cluster context\n", c.Name)
```

Comment:
This is good. Outputting the name of the hosting cluster context would be better. Does not have to be changed in this PR.